metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Installs and configures OpenLDAP (slapd) an open source implementation of LDAP.'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '3.1.1'
+version           '3.1.2'
```

Comment:
please don't update version information in metadata. we'll do this when we do the release. thanks.